recipe/common.php

```diff
@@ -238,8 +238,22 @@
                 run("$sudo chmod +a \"`whoami` allow delete,write,append,file_inherit,directory_inherit\" $dirs");
             } elseif (commandExist('setfacl')) {
                 if (!empty($httpUser)) {
-                    run("$sudo setfacl -R -m u:\"$httpUser\":rwX -m u:`whoami`:rwX $dirs");
-                    run("$sudo setfacl -dR -m u:\"$httpUser\":rwX -m u:`whoami`:rwX $dirs");
+                    if (!empty($sudo)) {
+                        run("$sudo setfacl -R -m u:\"$httpUser\":rwX -m u:`whoami`:rwX $dirs");
+                        run("$sudo setfacl -dR -m u:\"$httpUser\":rwX -m u:`whoami`:rwX $dirs");
+                    } else {
+                        // If run without sudo, check each dir
```

Comment:
@elfet 
On first deploy:
```
setfacl -R -m u:"apache":rwX -m u:`whoami`:rwX app/webroot/images
```
Then, apache creates files:
```
$ ll current/app/webroot/images/
total 112
-rw-r--r--. 1 apache apache 34821 Aug 21 17:07 1440144423_main.png
-rw-r--r--. 1 apache apache 34821 Aug 21 17:07 1440144423.png
-rw-rw-r--+ 1 apache apache 34821 Aug 21 17:07 1440144441_55d6dc39ce799.png
```

Next, exec setfacl on next deploy will throw runtime exception:
```
Run: cd /var/www/html/project/releases/20150821170801 &&  setfacl -R -m u:"apache":rwX -m u:`whoami`:rwX app/webroot/images

  Unable to setup correct permissions for writable dirs.
  You need co configure sudo's sudoers files to don't prompt for password,
  or setup correct permissions manually.

  [RuntimeException]
  setfacl: app/webroot/images/1440144441_55d6dc39ce799.png: Operation not permitted
  setfacl: app/webroot/images/1440144423_main.png: Operation not permitted
  setfacl: app/webroot/images/1440144423.png: Operation not permitted
```

I mentioned this problem here: https://github.com/deployphp/deployer/issues/433#issuecomment-133388894

Do you need more comments in the code?